### ISSUE TITLE
Use alt text as a fallback on image links that are hidden from screen readers

### ISSUE THREAD

**Issue by alice-blue** [iftechfoundation/ifdb#1388]
It was suggested that, even if we use aria-hidden="true" and tabindex="-1" on a redundant <a> element (which should result in any image inside the <a> element being hidden from screen readers), we should still use alt text on the image itself: "I think you should always include alt text on images that are links, even if they're hidden, just in case the user's browser is a bit naff and doesn't honour that."

So I guess if aria-hidden is not honored, the alt text would be a fallback, and the redundant link would get announced, but at least the announcement would be regular words and not a URL or filename or whatever the screen reader tries to substitute when it comes across an undescribed image link.

I suppose the other advantage is that if a sighted person has images turned off in the browser, the alt text would appear in the browser in the spot where the image should be, instead of a blank broken image.

It makes sense to me, though I don’t think I’ve come across this exact recommendation (using alt text in conjunction with aria-hidden). Does anyone know if there is official guidance on this?

This could affect https://github.com/iftechfoundation/ifdb/pull/1376 and https://github.com/iftechfoundation/ifdb/pull/1381.
